### PR TITLE
[GC] Release Spark metadata client v0.70.1

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: clients/spark/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
```
Comment (nopcoder) [inline clients/spark/CHANGELOG.md:2]: 0.7.1

**Review by nopcoder — CHANGES_REQUESTED** [review]
need the right semver

**Review by N-o-Z — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks! Unrelated test failure (not a Spark issue, and probably flaky) but rerunning because iirc we had a recent change on gateway MPU.

**Review by nopcoder — APPROVED** [review]
LGTM

**Pull request merged**